### PR TITLE
HDDS-3891. Add the usage of ofs in doc.

### PR DESCRIPTION
**Pull request by captainzmc**
## What changes were proposed in this pull request?

HDDS-2665 has been merged into the master. Currently, ozone supports ofs, so some ofs  usage needs to be added to Doc.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3891

## How was this patch tested?

update doc no need test.

### CODE REVIEW

File: hadoop-hdds/docs/content/interface/OzoneFS.md [hunk 1]
```diff
@@ -113,3 +113,50 @@ hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:6789/key
 Note: Only port number from the config is used in this case, 
 whereas the host name in the config `ozone.om.address` is ignored.
 
+## Setting up the ofs
+Please add the following entry to the core-site.xml.
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/OzoneFS.md:117]: These sections seem to overlap with #1144. We can simply add a reference here.

File: hadoop-hdds/docs/content/interface/OzoneFS.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ summary: Hadoop Compatible file system allows any application that expects an HD
 
 The Hadoop compatible file system interface allows storage backends like Ozone
 to be easily integrated into Hadoop eco-system.  Ozone file system is an
-Hadoop compatible file system.
+Hadoop compatible file system. Currently, Ozone supports two scheme: o3fs and ofs.
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/OzoneFS.md:26]: Can we add some highlights for the difference between the two, like you have mentioned in Line 147.

File: hadoop-hdds/docs/content/interface/OzoneFS.md [hunk 3]
```diff
@@ -113,3 +113,50 @@ hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:6789/key
...
+  <name>fs.ofs.impl</name>
+  <value>org.apache.hadoop.fs.ozone.RootedOzoneFileSystem</value>
+</property>
+<property>
+  <name>fs.defaultFS</name>
+  <value>ofs://om-host.example.com/</value>
+</property>
+{{< /highlight >}}
+
+This will make all the volumes and buckets to be the default file system for HDFS dfs commands and register the o3fs file system type.
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/interface/OzoneFS.md:130]: Can we change " default file system for HDFS dfs commands " =>  " default Hadoop compatible file system"

File: hadoop-hdds/docs/content/interface/OzoneFS.md (hunk 2 above)
Comment (captainzmc) [reply hadoop-hdds/docs/content/interface/OzoneFS.md:26]: The main difference between the two has been added.

File: hadoop-hdds/docs/content/interface/OzoneFS.md (hunk 1 above)
Comment (captainzmc) [reply hadoop-hdds/docs/content/interface/OzoneFS.md:122]: References to ofs.md have been added.

**Comment by captainzmc** [comment]
Thanks @xiaoyuyao for the review. I have fixed doc. Please take another look.

**Pull request merged**